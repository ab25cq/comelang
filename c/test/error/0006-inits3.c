/*
PATTERN: overlap
PATTERN: c:6:
*/

int arr[3] = {[0] = 1, [1] = 2, [0] = 3};

