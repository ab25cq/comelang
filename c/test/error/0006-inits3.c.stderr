fields in init overlaps with another field at test/error/0006-inits3.c:6:21
int arr[3] = {[0] = 1, [1] = 2, [0] = 3};
                    ^
